### ISSUE TITLE
Vscode plugin not starting on windows, had to add to path variable.

### ISSUE THREAD

**Issue by MichaelMorozovCodit** [daveshanley/vacuum#479]
I got an error when setting up the vscode extension:

```
[Error - 3:03:59 PM] Starting client failed
Launching server using command vacuum.cmd failed.
```

So I figured it had to be because it's not added to PATH on Windows when installing through npm. Thought you should know. I ended up creating a symlink explicitly called `vacuum.cmd` in order for it to work, which it now does.

**Comment by LasneF** [comment]
it looks strange  , are you sure you have install it globally ? with -g option 
i mean if you open a shell it should run wherever you are , and so it is for the vscode 
it might requires to restart  a terminal ,

**Comment by MichaelMorozovCodit** [comment]
Yes, I ran the `npm install -g @quobix/vacuum` command like in the description.

**Comment by MichaelMorozovCodit** [comment]
Maybe related to this, the extension doesn't start automatically every time I open up VScode.

**Comment by daveshanley** [comment]
This has been addressed. 

https://open-vsx.org/extension/pb33f/vacuum
https://marketplace.visualstudio.com/items?itemName=pb33f.vacuum
